index.html

```diff
@@ -0,0 +1,110 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+
+    <title>Intuit Accounts - Sign Up</title>
+
+    <link rel="stylesheet" href="css/style.css">
+</head>
+
+<body>
+
+    <header>
+        <div class="header-section">
+            <div class="signin-container">
+                <div class="signin-text">
+                    <p class="p-signin">Already have an Intuit account?</p>
+                    <a href="#">I forgot my User ID or Password</a>
+                </div>
+                <div>
+                    <button class="signin-button">
+                        Sign In
+                    </button>
+                </div>
+            </div>
+            <div class="logo-container center-div">
+                <a href="#"><img src="images/intuit-logo.png" alt="Intuit Logo"></a>
+            </div>
+            <div class="services-container center-div">
+                <a href="#" class="serviceIcon"><img src="images/turboIcon.png" alt="Turbotax Logo"></a>
+                <a href="#" class="serviceIcon"><img src="images/quickIcon.png" alt="Quickbooks Logo"></a>
+                <a href="#" class="serviceIcon"><img src="images/mintIcon.png" alt="Mint Logo"></a>
+            </div>
+        </div>
+    </header>
+
+    <main>
+
+        <section>
+            <form name="create-account">
+                <fieldset class="form-container">
+                    <div class="header-container">
+                        <div class="form-header">
+                            <h1>Create an Intuit account</h1>
+                            <p>One account for everything Intuit, including Mint. </p>
+                            <a href="#">Learn More</a>
+                        </div>
+                    </div>
+                    <br>
+
+
+                    <label for="email">
+                        Email address
+                    </label>
+                    <div class="input-cont">
+                        <input type="email" name="email" id="email" placeholder="You'll verify this later" required>
```

Comment:
`AWESOME`

Good job associating the `<label>` with an `<input>` element. :clap: :+1: 

> #### Associating a `<label>` with an `<input>` element offers some major advantages:
> 
> - The label text is not only visually associated with its corresponding text input; it is programmatically associated with it too. This means that, for example, a screen reader will read out the label when the user is focused on the form input, making it easier for an assistive technology user to understand what data should be entered.
> 
> - You can click the associated label to focus/activate the input, as well as the input itself. This increased hit area provides an advantage to anyone trying to activate the input, including those using a touch-screen device.

Alternatively, you can nest the `<input>` directly inside the `<label>` like this: 

``` html
  <label for="email">Email address <br>
       <input type="email" name="email" id="email">
  </label> 
```


You can learn more [here](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/label)